modules/sqs/main.tf

```diff
@@ -53,17 +53,81 @@ resource "aws_sqs_queue" "metadata_queue" {
 ## ====================================================================================================
 resource "aws_sqs_queue" "staged_recovery_queue" {
   ## OPTIONAL
-  name                        = "${var.prefix}-staged-recovery-queue.fifo"
-  fifo_queue                  = true
-  content_based_deduplication = true
-  delay_seconds               = var.sqs_delay_time_seconds
-  max_message_size            = var.sqs_maximum_message_size
-  message_retention_seconds   = var.staged_recovery_queue_message_retention_time_seconds
-  tags                        = local.tags
-  policy                      = data.aws_iam_policy_document.staged_recovery_queue_policy.json
-  visibility_timeout_seconds  = 1800 # Set to double lambda max time
+  name                       = "${var.prefix}-staged-recovery-queue"
+  delay_seconds              = var.sqs_delay_time_seconds
+  max_message_size           = var.sqs_maximum_message_size
+  message_retention_seconds  = var.staged_recovery_queue_message_retention_time_seconds
+  tags                       = local.tags
+  policy                     = data.aws_iam_policy_document.staged_recovery_queue_policy.json
+  visibility_timeout_seconds = 1800 # Set to double lambda max time
+  depends_on = [
+    aws_sqs_queue.staged_recovery_deadletter_queue
+  ]
+  redrive_policy = jsonencode({
+    deadLetterTargetArn = aws_sqs_queue.staged_recovery_deadletter_queue.arn,
+    maxReceiveCount     = 3 #number of times a consumer tries receiving a message from the queue without deleting it before being moved to DLQ.
+  })
 }
 
+resource "aws_sqs_queue" "staged_recovery_deadletter_queue" {
```

Comment:
may think of shortening from `staged_recovery_deadletter_queue` to `staged_recovery_dlq`

Reply:
updated.